Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -9,6 +9,7 @@
 
 import java.util.List;
 
+@Transactional
```

Comment:
Why deleteing this from methods and putting only to class? Any specific purpose?

Reply:
@EvilXa0S  If you mark a class with @Transactional, all methods of that class become @Transactional.